### ISSUE TITLE
CMS 4.12: Saving changed field inside ui-accordions throws error on the console

### ISSUE THREAD

**Issue by lerni** [silverstripe/silverstripe-admin#1441]
With `1.12.1` an error is thrown on console like `Uncaught Error: no such method 'activate' for accordion widget instance` when saving a page after changing a field in `Root.Main.Metadata` or most probable with all ui-accordions.

## Acceptance criteria
- Validate that the issue can be replicated.
- Warning has been squashed.

### PR
- https://github.com/silverstripe/silverstripe-admin/pull/1472

**Comment by maxime-rainville** [comment]
@lerni Can you confirm if this is only a console warning? Does it actually prevent you from saving the page?

**Comment by lerni** [comment]
I’ve tried to reproduce it again and initially saving values succeeded. After a few more tries I’ve seen it fail again, but it behaves weird. It has occurred on a vanilla instance but right now I cannot reproduce it constantly, so I’m posting the recording that I made from a customized version - value was not updated after reloading, also why is .cms-navigator rendered wrong at the same time?  
![ezgif com-video-to-gif(1)](https://user-images.githubusercontent.com/472115/220162371-bc5261c9-fb0f-49c3-9695-f21d1637fc4e.gif)

**Comment by sabina-talipova** [comment]
Issue still exists. It has been tested in `recipe-kitchen-sink: 4` and `recipe-kitchen-sink: 5`.

**Comment by sabina-talipova** [comment]
The jQuery team has removed the `activate` event in v1.10 and recommend to use `active` option instead.
https://jqueryui.com/upgrade-guide/1.10/#removed-activate-method-use-active-option
https://jqueryui.com/upgrade-guide/1.9/#deprecated-activate-method

**Comment by chrispenny** [comment]
Hi team,

I'm experiencing this error in version `2.0.0-beta1`, but the Site Tree also disappears.

## Video

https://user-images.githubusercontent.com/505788/225193611-8de05685-defe-4327-b016-68876a69c12c.mov

## Project set up

`composer.json`:
```json
"require": {
    "php": "^8.1",
    "ext-simplexml": "*",
    "ext-libxml": "*",
    "silverstripe/recipe-cms": "^5",
    "silverstripe/auditor": "^3",
    "silverstripe/crontask": "^3",
    "silverstripe/dynamodb": "^5",
    "silverstripe/environmentcheck": "^3",
    "silverstripe/raygun": "^4",
    "silverstripe/totp-authenticator": "^5",
    "wilr/silverstripe-googlesitemaps": "dev-main"
},
```

* Standard `Page` with no customisations (no elemental, etc).
* We do have a custom `editor.css` file, but I (effectively) emptied it.

## Console logs

```
Uncaught Error: no such method 'activate' for accordion widget instance
    at Function.error (vendor.js?m=1675732377:1:849397)
    at HTMLDivElement.<anonymous> (vendor.js?m=1675732377:1:104794)
    at Function.each (vendor.js?m=1675732377:1:849777)
    at e.fn.init.each (vendor.js?m=1675732377:1:848256)
    at e.fn.<computed> [as accordion] (vendor.js?m=1675732377:1:104677)
    at e.fn.init.restoreFieldFocus (bundle.js?m=1675732377:1:239018)
    at o (vendor.js?m=1675732377:1:83471)
    at e.fn.init.restoreFieldFocus (vendor.js?m=1675732377:1:83667)
    at e.fn.init.ontabstaterestored (bundle.js?m=1675732377:1:238040)
    at e.fn.init.<anonymous> (vendor.js?m=1675732377:1:91858)
```

**Comment by GuySartorelli** [comment]
@chrispenny I've just tried your project locally and did a hotfix based on https://github.com/silverstripe/silverstripe-admin/issues/1441#issuecomment-1469192978 and can confirm it's the same issue. Not sure why the tree disappears with CMS 5 but when a PR has been raised and merged replacing use of `'activate'` method with `'active'` option it'll fix both the original and this new presentation of the bug.

**Comment by GuySartorelli** [comment]
This has now been fixed - tagged in CMS 4 as [1.12.4](https://github.com/silverstripe/silverstripe-admin/releases/tag/1.12.4) and in CMS 5 as [2.0.0-beta2](https://github.com/silverstripe/silverstripe-admin/releases/tag/2.0.0-beta2)

**Comment by chrispenny** [comment]
Thanks, team! Much appreciated!
